server/docs/source/index.html.md

```diff
@@ -582,6 +582,10 @@ to | The end date and time (example: `?to=2017-01-01T13:15:00.00Z`) | now
   "version": "0.1.0",
   "registry": "https://stack-registry.kontena.io",
   "expose": "peer",
+  "parent": { "name": "parent-stack-name" },
+  "children": [
+    { "name": "child-stack-name" }
```

Comment:
Should parent/children objects also have a stack id?

Reply:
Currently it goes like:

```ruby
json.parent stack.has_parent? ? { name: stack.parent_name } : nil
json.children stack.children.pluck(:name).map { |child_name| { name: child_name } }
```

Does the id have any significance? I don't think you can't use it for anything in the API, the `stack.to_path` which is used as "id" in the docs comes from `"#{self.grid.try(:name)}/#{self.name}"`, so the JSON could just become:

```ruby
json.parent stack.has_parent? ? { name: stack.parent_name, id: "#{stack.grid.try(:name)}/#{stack.parent_name}" } : nil
json.children stack.children.pluck(:name).map { |child_name| { id: "#{stack.grid.try(:name)}/#{child_name}", name: child_name } }
```